pyanaconda/modules/network/network.py

```diff
@@ -61,7 +61,7 @@ def __init__(self):
         self._firewall_module = FirewallModule()
 
         self.hostname_changed = Signal()
-        self._hostname = "localhost.localdomain"
+        self._hostname = None
```

Comment:
Could you please change it to an empty string? `None` is not a valid DBus value and it can lead to errors.